packages/slice-machine/server/src/api/versions.ts

```diff
@@ -0,0 +1,27 @@
+import { hasYarn, checkVersion } from "@lib/utils/version";
+import { name } from "../../../package.json";
```

Comment:
I think we can defined a name in a const instead of doing that. Package.json shouldn't be used in runtime